### ISSUE TITLE
Pesticide filters may not be showing all records

### ISSUE THREAD

**Issue by amyburness** [kartoza/django-bims#3290]
**Describe the bug**
The pesticide filters, once run only show a small subset of all of the sites when the layer covers all of South Africa as below. 

**To Reproduce**
1. Go to filters and select all of the categories for Algae risk under spatial filters
2. Click on apply
3. view the number of sites. 
4. See the small number of sites returned.

**Expected behavior**
I expect data values for more sites as there are 19,482 records on FBIS, and the algae layer covers most of the South Africa area. 

**Screenshots**
![image](https://user-images.githubusercontent.com/77841514/208443618-954a3f17-af53-4148-a4c9-fdc717b098bf.png)

**Comment by amyburness** [comment]
@dimasciput I remember a similar issue in the past is there a fix?

**Comment by dimasciput** [comment]
This should be fixed now.

**Comment by helendallas** [comment]
@amyburness Please can you check and close if working

**Comment by amyburness** [comment]
@helendallas, yes, this looks better it is pulling out the expected number of points. You can close.
![image](https://user-images.githubusercontent.com/77841514/219053451-a8320392-5080-4af5-9224-247bdd56137b.png)
